tests/wicked/aggregate/sut/t17_teaming_ab_ethtool.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Teaming, Active-Backup Ethtool
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use Mojo::Base 'wickedbase';
+use testapi;
+
+
+sub run {
+    my ($self, $ctx) = @_;
+    record_info('INFO', 'Teaming, Active-Backup Ethtool');
+    $self->setup_team('ab-ethtool', $ctx->iface(), $ctx->iface2());
+    $self->validate_interfaces('team0', $ctx->iface(), $ctx->iface2(), 0);
+    my $active_link1 = $self->get_team_active_link('team0');
+    record_info('INFO', "Active Link is $active_link1. Bringing it down...");
```

Comment:
you already posted output of teamdctl while getting active link . this output contains active link already , right ? why need to print it one more time ? I think it us too much

Reply:
ok, where would you like to keep the message? In get_team_active_link or in the test code?

Reply:
yap , the one inside `get_team_active_link` :
1. contains more info ( full teamdctl output not just active link name ) 
2. don't need to copy/paste same thing after each `get_team_active_link` :)

Reply:
fixed